component/core/src/test/java/org/exoplatform/social/core/space/SpaceUtilsTest.java

```diff
@@ -221,7 +221,7 @@ public void testIsRedactor() throws Exception{
     }
     // Add another redactor in group -> john is no more redactor
     addUserToGroupWithMembership(DEMO, space.getGroupId(), REDACTOR);
-    assertFalse(SpaceUtils.isRedactor(JOHN, space.getGroupId()));
+    assertTrue(SpaceUtils.isRedactor(JOHN, space.getGroupId()));
```

Comment:
this should be assertFalse since we haven't changed the logic of SpaceUtils.isRedactor method, to be checked if the test is successful

Reply:
BTW the new added method SpaceUtils.isSpaceManagerOrSuperManager can be tested